src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/ruleNode.tsx

```diff
@@ -266,17 +266,7 @@ class RuleNode extends React.Component<Props> {
       case MailActionTargetType.Team:
         return null;
       case MailActionTargetType.Member:
-        return (
```

Comment:
do we no longer need this warning message?

Reply:
@robinrendle said "i dare you to remove that useless warning message"